tests/src/JIT/Performance/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,38 @@
+@setlocal
```

Comment:
I doubt that we want to run actual benchmarks when running test.   We do fire up the benchmark harness, it does nothing, and returns.    The harness is built as a test to avoid all the issues with having separate tool and build process.    If we create a utility script directory, with associated project.json for all our utilities and build it as part of standard build, then we can move harness there.